### ISSUE TITLE
[Request] Data upload to S3 Bucket?

### ISSUE THREAD

**Issue by wxmann** [CSSEGISandData/COVID-19#1778]
Right now, if I'd like to build a dashboard or some other visualization of COVID-19, I need to `git pull` this repo once a day to get the latest information. Is there a way this information can be uploaded to a publicly accessible storage location such as an S3 bucket? This would also open the data to researchers in the future without coupling to this Github account.

Many thanks and regards. Thank you for all who maintain this dataset!

**Comment by karx** [comment]
@wxmann Github provides great version control for this kind of dataset and helps encourage collaboration.

Can you let me know were you planning to pull the data from S3. I would be able to help you at doing the same, keeping the source as GitHub.

If you really do want it to be on S3, let me know, we can set up a small pipeline to create a copy in S3 whenever this github repository is updated. It would be redundant tho.

Also, have you checked out this repo: https://github.com/summyfeb12/COVID-19-JHU-Data-API

Cheers.

**Comment by sengelbert** [comment]
I/We would definitely use data pushed to S3 as well

**Comment by cipriancraciun** [comment]
I can point you to my derived dataset (described in #1281 and) available at https://github.com/cipriancraciun/covid19-datasets, for which I also publish the data on "the web" accessible via plain HTTP.  (See the description of that issue, or the repository on how to fetch the data.  The data is hosted on Netlify and served through CloudFlare, thus load shouldn't be an issue.)

**Comment by wxmann** [comment]
@karx and @cipriancraciun thank you for your responses and pointing out to me those repos. Let me see if they help with my use cases.

**Comment by martin-majlis** [comment]
@wxmann : You can access the raw data directly from the GitHub without need to pull them.

1. Go to your favorite data source - https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_confirmed_global.csv
2. Click on "Raw"
3. You will get here - https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_confirmed_global.csv
4. And you have the latest data :)

**Comment by oltdaniel** [comment]
@wxmann You hve the GitHub CDn on your hand, S3 is not useful here. Please close this one, due to useless feature request.

**Comment by sengelbert** [comment]
Actually I would disagree, sharing to S3 is not "useless". Many companies and organizations share open datasets on S3 (and other places) for others to utilize (https://aws.amazon.com/opendata/public-datasets/).

**Comment by cipriancraciun** [comment]
@sengelbert the main issue with S3 (and other similar solutions) is who pays for the storage and network bandwidth?  (One common solution with AWS S3 for datasets is to ask the "user" to pay for the bandwidth, but now the user needs to have an AWS account.)

That is why GitHub is the best and cheapest solution.  You don't need anything besides a GitHub account, and everything is free.  (But I'm certain there are some usage limits...)

For example one could alternatively use Netlify or GitHub Pages, but both limit the bandwidth to 100 GiB per month.  However if you throw CloudFlare in front of that, you basically get almost unlimited bandwith for free.  (But now you are required to have your DNS at CloudFlare, and things get complicated.)

**Comment by sengelbert** [comment]
@cipriancraciun Yeah I think that is a fair and a reasonable answer. I completely understand the costing and lock-in points.

I think my only point was with the rise of dataset sharing services (Kaggle, AWS, Data World, etc) requesting to have a great dataset like this standardized and shared within them is not a "useless" request. 

Thanks for all your work on this

**Comment by hannesrauhe** [comment]
For anyone who somehow ends up here. Amazon provides a publicly readable bucket with related data:

https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/

At first glance, this GH repo contains more data. Maybe they can mirror this to their bucket. Will try to reach out for them.
